doc/source/libraries/xorbits_datasets/index.rst

```diff
@@ -0,0 +1,27 @@
+.. _xorbits_datasets_index:
+
+
+================
+Xorbits Datasets
```

Comment:
How about calling it Xorbits data? Since we may have a sole sub library called xorbits.datasets.